account_move_line_tax_editable/views/account_move.xml

```diff
@@ -9,7 +9,7 @@
         <field name="model">account.move</field>
         <field name="inherit_id" ref="account.view_move_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='line_ids']/tree/field[@name='name']" position="after">
+            <xpath expr="//field[@name='invoice_line_ids']/tree/field[@name='name']" position="after">
```

Comment:
Invoice lines? This should be on journal items.

Reply:
Well, in v13 there are two pages, one for invoice_line_ids and another for line_ids. I checked that the old page of line_ids in v12 is the invoice_line_ids for v13 (comparing the fields in the page). The one page new is for the line_ids, that show a different list of fields.

Reply:
They are in an incorrect position:

![Selección_026](https://user-images.githubusercontent.com/7165771/73550285-9797e300-4444-11ea-81ec-00d183b75ff9.png)

And this is only valid when you are looking to journal items, not to invoices. Even more, I would say that they should be hidden if the move is an invoice.

Reply:
haha ok

Reply:
can you check now?